packages/next/src/server/config.ts

```diff
@@ -713,6 +713,9 @@ function assignDefaults(
           'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/sparkles!lucide-react',
         '(TextSelection|LucideTextSelection|TextSelectionIcon)':
           'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/icons/text-select!lucide-react',
+        // Alias for createLucideIcon
+        '(createLucideIcon)':
+          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/createLucideIcon!lucide-react',
```

Comment:
Great catch, then we have nearly everything, the only thing that is missing now is `icons`.
```js
import * as index from './icons/index.js';
export { index as icons };
```


```suggestion
        // Alias for createLucideIcon
        '(createLucideIcon)':
          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/createLucideIcon!lucide-react',
        // Alias for icons
        '(icons)':
          'modularize-import-loader?name={{ member }}&from=default&as=default&join=../esm/index!lucide-react',
```